CONTRIBUTING.rst

```diff
@@ -1466,6 +1466,9 @@ We don't create new issues on JIRA anymore. The reason we still look at JIRA iss
    * newbie questions (#newbie-questions channel)
```

Comment:
This channel does no longer exist

Reply:
may I know please why we archived that channel? :)

Reply:
low traffic and most of times cross-posting in random or troubleshooting